### DISCUSSION TITLE
Superscript in thesis cover title

### DISCUSSION THREAD

**Discussion by luisacbscs** [joaomlourenco/novathesis#451] [Q&A] [answered]
Hi all,

I was wondering if anyone knows how to include superscript text in the thesis cover title (more specifically, ¹⁸F). I generally use either \textsuperscript or math mode (e.g. \ensuremath{{}^{\text{18}}}F). However, I get an error when using either (in math mode the problem is with the \text command). Instead of \text I tried using \mbox, which worked for another problem I was having (the title also didn't let me include square brackets). Although \mbox maintains the font family, it also maintains the font size (which does not work for me since I need it for superscript text). Currently I am using math mode without the \text command, but the font is the math mode font (and I'd like it to be the title's font). Attached are illustrations of the problems (using \mbox and the font being too large, and using pure math mode and the font family changing to the default math font).

Thank you in advance for your help!
<img width="56" alt="mbox" src="https://github.com/joaomlourenco/novathesis/assets/100357143/7f487ae5-62c1-4ad2-a29a-eb1e29944c5c">
<img width="59" alt="math_font" src="https://github.com/joaomlourenco/novathesis/assets/100357143/762af690-17f4-45e7-be59-76ef435ec08c">

**Comment by joaomlourenco** [comment]
Can you please try `\textsuperscript{18}F` and let me know if it worked for you?

**Reply by luisacbscs** [reply]
Hi,

Sorry, I mentioned \textsuperscript ran into an error but did not fully explain it. When I try `\textsuperscript{18}F` I get the error message: `Incomplete \iffalse; all text was ignored after line 19.`, given that line 19 is where I have the title command (`\nttitle`). That's why I resorted to using math mode.

Thank you for your help!

**Comment by joaomlourenco** [comment, marked as answer]
Can you please try this title
```latex
\nttitle(main,en){A Very Long and Impressive\\Thesis Title $^{\text{18}}$F with \ensuremath{{}^{\text{18}}}F a Forced Line Break}
```
It works for me… and the second “instance” is your own definition.

If this one does not work for you, then update your LaTeX distribution to the latest version.
If this one works for you but yours still don't, please copy/paste here your title and I'll try it myself.

**Reply by luisacbscs** [reply]
I have been using version 6.9.5 as 6.9.10 results in a few problems for me, and in 6.9.5 that solution does not work (it does in the latest version so thank you). I didn't think it would depend on the version and I'm sorry for the confusion. Thank you very much and I'll try and manage.

**Reply by joaomlourenco** [reply]
So, if you are "upgrading" to the latest version, please be aware that some things changes in the format of the config files (e.g., some option names).
Please go step by step in the upgrade, fixing the errors as they come, to reduce the probability of having the errors accumulating and making your debugging process really hard.
